record/Student.java

```diff
@@ -0,0 +1,17 @@
+package record;
+
+public record Student(String name, int grade, int classNumber) {
+
+    public Student(String name, int classNumber) {
+        this(name, Constants.NINTH_GRADE, classNumber);
```

Comment:
```suggestion
        this(name, Constants.NINTH_GRADE, classNumber);
```
same here about the ninth as i said in 10 12